internal/provider/jwt_issuers.go

```diff
@@ -109,195 +106,190 @@ func (r *apiOidcConfigResource) Configure(
 	ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse,
 ) {
-	var state ApiOidcConfig
+	var state JWTIssuer
 	diags := req.State.Get(ctx, &state)
 	resp.Diagnostics.Append(diags...)
 	if resp.Diagnostics.HasError() {
 		return
 	}
 
-	traceAPICall("DeleteApiOidcConfig")
-	//nolint:staticcheck
-	_, _, err := r.provider.service.DeleteApiOidcConfig(ctx, state.ID.ValueString())
+	traceAPICall("DeleteJWTIssuer")
+	_, _, err := r.provider.service.DeleteJWTIssuer(ctx, state.ID.ValueString())
 	if err != nil {
 		resp.Diagnostics.AddError(
-			"Error deleting API OIDC Config",
-			fmt.Sprintf("Could not delete API OIDC Config: %s", formatAPIErrorMessage(err)),
+			"Error deleting JWT Issuer",
+			fmt.Sprintf("Could not delete JWT Issuer: %s", formatAPIErrorMessage(err)),
 		)
 		return
 	}
 
-	// Remove resource from state
 	resp.State.RemoveResource(ctx)
 }
 
-func (r *apiOidcConfigResource) ImportState(
-	ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse,
-) {
-	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
-}
-
-func NewApiOidcConfigResource() resource.Resource {
-	return &apiOidcConfigResource{}
-}
-
-func loadApiOidcConfigToTerraformState(
-	apiOidcConfig *client.ApiOidcConfig, state *ApiOidcConfig,
+func loadJWTIssuerToTerraformState(
+	jwtIssuer *client.JWTIssuer, state *JWTIssuer,
 ) {
-	state.ID = types.StringValue(apiOidcConfig.Id)
-	state.Audience = types.StringValue(apiOidcConfig.Audience)
-	state.Issuer = types.StringValue(apiOidcConfig.Issuer)
-	state.Jwks = types.StringValue(apiOidcConfig.Jwks)
-	state.Claim = types.StringPointerValue(apiOidcConfig.Claim)
-	state.IdentityMap = identityMapToTerraformState(apiOidcConfig.IdentityMap)
+	state.ID = types.StringValue(jwtIssuer.Id)
+	state.Audience = types.StringValue(jwtIssuer.Audience)
+	state.IssuerURL = types.StringValue(jwtIssuer.IssuerUrl)
+	state.Jwks = types.StringPointerValue(jwtIssuer.Jwks)
+	state.Claim = types.StringPointerValue(jwtIssuer.Claim)
+	state.IdentityMap = identityMapToTerraformState(jwtIssuer.IdentityMap)
 }
 
-func identityMapFromTerraformState(identityMap *[]IdentityMapEntry) *[]client.ApiOidcIdentityMapEntry {
+func identityMapFromTerraformState(identityMap *[]IdentityMapEntry) *[]client.JWTIssuerIdentityMapEntry {
 	if identityMap == nil {
 		return nil
 	}
-	var out []client.ApiOidcIdentityMapEntry
+	var out []client.JWTIssuerIdentityMapEntry
 	for _, mapEntry := range *identityMap {
-		out = append(out, client.ApiOidcIdentityMapEntry{
-			CcIdentity:    mapEntry.CcIdentity.ValueStringPointer(),
-			IsRegex:       mapEntry.IsRegex.ValueBoolPointer(),
+		out = append(out, client.JWTIssuerIdentityMapEntry{
 			TokenIdentity: mapEntry.TokenIdentity.ValueStringPointer(),
+			CcIdentity:    mapEntry.CcIdentity.ValueStringPointer(),
+			// TODO: Need to remove once the field has been removed from the API.
+			IsRegex: types.BoolValue(false).ValueBoolPointer(),
```

Comment:
Do we need this?  Can't we just leave it empty?

Reply:
Because we did not remove the field from the API yet, the response still contains this field with a default value (`false`).